.github/workflows/update-pr-metadata.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Add assignees
         if: |
-          github.event_name == 'pull_request' &&
+          startsWith(github.event_name, 'pull_request') &&
```

Comment:
nit: Or maybe `github.event_name == 'pull_request_target'`?

Reply:
you are right.. since the `pull_request` family is big lol

Reply:
actually this line is not necessary anymore.. used to label issues as well. but not needed now